docs/supported-features.md

```diff
@@ -0,0 +1,288 @@
+https://caniuse.com/#feat=mediasource
+
+Note that browsers with native HLS support may play content with the native player, unless
+the [overrideNative] option is used. Some notable browsers with native HLS players are:
+
+* Safari (macOS and iOS)
+* Chrome Android
+* Firefox Android
+
+However, due to the limited features offered by some of the native players, the only
+browser on which VHS defaults to using the native player is Safari (macOS and iOS).
+
+## Streaming Formats and Media Types
+
+### Streaming Formats
+
+VHS aims to be mostly streaming format agnostic. So long as the manifest can be parsed to
+a common JSON representation, VHS should be able to play it. However, due to some large
+differences between the major streaming formats (HLS and DASH), some format specific code
+is included in VHS. If you have another format you would like supported, please reach out
+to us (e.g., file an issue).
+
+* [HLS] (HTTP Live Streaming)
+* [MPEG-DASH] (Dynamic Adaptive Streaming over HTTP)
+
+### Media Container Formats
+
+* [TS] (MPEG Transport Stream)
+* [MP4] (MPEG-4 Part 14: MP4, M4A, M4V, M4S, MPA), ISOBMFF
+* [AAC] (Advanced Audio Coding)
+
+### Codecs
+
+If the content is packaged in an [MP4] container, then any codec supported by the browser
+is supported. If the content is packaged in a [TS] container, then the codec must be
+supported by [the transmuxer]. The following codecs are supported by the transmuxer:
+
+* [AVC] (Advanced Video Coding, h.264)
+* [AVC1] (Advnced Video Coding, h.265)
+* [HE-AAC] (High Efficiency Advanced Audio Coding, mp4a.40.5)
+* LC-AAC (Low Complexity Advanced Audio Coding, mp4a.40.2)
+
+## General Notable Features
+
+The following is a list of some, but not all, common streaming features supported by VHS.
+It is meant to highlight some common use cases (and provide for easy searching), but is
+not meant serve as an exhaustive list.
+
+* VOD (video on demand)
+* LIVE
+* Multiple audio tracks
+* Timed [ID3] Metadata is automatically translated into HTML5 metedata text tracks
+* Cross-domain credentials support with [CORS]
+* Any browser supported resolution (e.g., 4k)
+* Any browser supported framerate (e.g., 60fps)
+* [DRM] via [videojs-contrib-eme]
+* Audio only
+* Video only
+* In-manifest [WebVTT] subtitles are automatically translated into standard HTML5 subtitle
+  tracks
+* [AES-128] and SAMPLE-AES segment encryption
+
+## Notable Missing Features
+
+Note that the following features have not yet been implemented or may work but are not
+currently suppported in browsers that do not rely on the native player. For browsers that
+use the native player (e.g., Safari for HLS), please refer to their documentation.
+
+### Container Formats
+
+* [WebM]
+* [WAV]
+* [MP3]
+* [OGG]
+
+### Codecs
+
+If the content is packaged within an [MP4] container and the browser supports the codec, it
+will play. However, the following are some codecs that are not routinely tested, or are not
+supported when packaged within [TS].
```

Comment:
I think HEVC would be another one on this list? HLS specifies that HEVC must be packaged in an mp4.

Reply:
I think we should be good for support then, aren't we?